genie-web/src/test/java/com/netflix/genie/web/security/oauth2/pingfederate/PingFederateRemoteTokenServicesUnitTests.java

```diff
@@ -156,8 +156,7 @@ public void canLoadAuthentication() {
             scope1 + " " + scope2
         );
 
-        @SuppressWarnings("unchecked")
-        final ResponseEntity<Map> response = Mockito.mock(ResponseEntity.class);
+        @SuppressWarnings("unchecked") final ResponseEntity<Map> response = Mockito.mock(ResponseEntity.class);
```

Comment:
Nitpick: both Oracle and Google style guidelines say each annotation is on a new line

Reply:
IDEA must have done this for me. If you want to nitpick about it we should probably find a way to get our checkstyle to enforce this